packages/backend-data/API.md

```diff
@@ -35,8 +35,10 @@ export type DataProps = {
 // @public
 export type DataSchema = string | DerivedModelSchema;
 
+// Warning: (ae-forgotten-export) The symbol "AuthorizationModeMapping" needs to be exported by the entry point index.d.ts
+//
 // @public
-export type DefaultAuthorizationMode = 'AWS_IAM' | 'AMAZON_COGNITO_USER_POOLS' | 'OPENID_CONNECT' | 'API_KEY' | 'AWS_LAMBDA';
+export type DefaultAuthorizationMode = keyof typeof AuthorizationModeMapping;

```

Comment:
We should do something in implementation to not drag `AuthorizationModeMapping` into public types.

Reply:
Got it. I think splitting this string literal from the mapping and moving the mapping directly into where its used to translate from one auth mode structure to another speaks to both this comment and the above comment. I have made this change, will be up in a minute.

Reply:
Whatever it takes :)
Internal mechanism to map this can be changed with patch version release if types don't leak into public surface.